src/Server/User_Services.py

```diff
@@ -0,0 +1,183 @@
+import os
+import sys
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(project_root)
+from Models.UserLogin import User
+from Models.Recommendation_System import RecommendationEngine
+from Models.Admin import Admin
+from Models.chef import Chef
+from Models.Employee import Employee
+from Models.Menu import MenuManager
+import logging
+logging.basicConfig(filename='user_activity.log', level=logging.INFO, format='%(asctime)s - %(message)s')
+
+class User_Service:
```

Comment:
looks like whole code you have added in the switch cases only. It will be good if you moved the code to individual functions. easy to reuse.